mlflow/store/artifact/cloud_artifact_repo.py

```diff
@@ -212,20 +213,30 @@ def _parallelized_download_from_cloud(self, file_size, remote_file_path, local_p
             )
 
             if failed_downloads:
-                new_cloud_creds = self._get_read_credential_infos([remote_file_path])[0]
-                new_signed_uri = new_cloud_creds.signed_uri
-                new_headers = self._extract_headers_from_credentials(new_cloud_creds.headers)
+                num_retries = 3
                 for chunk in failed_downloads:
-                    _logger.warning(
-                        f"Retrying download of chunk {chunk.index} of {remote_file_path}"
-                    )
-                    download_chunk(
-                        range_start=chunk.start,
-                        range_end=chunk.end,
-                        headers=new_headers,
-                        download_path=local_path,
-                        http_uri=new_signed_uri,
-                    )
+                    new_cloud_creds = self._get_read_credential_infos([remote_file_path])[0]
+                    new_signed_uri = new_cloud_creds.signed_uri
+                    new_headers = self._extract_headers_from_credentials(new_cloud_creds.headers)
+                    _logger.warning(f"Retrying download of chunk {chunk} for {remote_file_path}")
+                    for retry in range(num_retries):
+                        try:
+                            _logger.warning(f"Attempt {retry + 1}/{num_retries}")
```

Comment:
```suggestion
                            _logger.debug(f"Attempt {retry + 1}/{num_retries}")
```

Drive-by comment, I was thinking when we ship this, should we use a lower severity log-level than `warning`? Maybe `debug` or `info`? Since retrying should be harmless/users shouldn't care most of the time

Reply:
And especially for R2 it seems reasonably likely you'll need at least one part retry in the process of downloading a large model

Reply:
+1 good idea. Otherwise based on the rates that you were seeing during testing, this could get REALLY chatty on stdout / stderr. `debug` seems appropriate, matching retry policies that we see in Spark for failed fetches during DataFrame loads.

Reply:
+1. Will make this change.

Reply:
Chose info. debug is not printed out (unless you modify the logging level)